pkg/webhook/workload/mutating/workload_update_handler.go

```diff
@@ -369,6 +386,22 @@ func (h *WorkloadHandler) handleCloneSet(newObj, oldObj *kruiseappsv1alpha1.Clon
 	} else if rollout == nil || rollout.Spec.Strategy.IsEmptyRelease() {
 		return false, nil
 	}
+	/*
+		continuous release (or successive release) is not supported for bluegreen release, especially for cloneset,
+		here is why:
+		suppose we are releasing a cloneset, which has pods of both v1 and v2 for now. If we release v3 before
+		v2 release is done, the cloneset controller might scale down pods without distinguishing between v1 and v2.
+		This is because our implementation is based on the minReadySeconds, pods of both v1 and v2 are "unavailable"
+		in the progress of rollout.
+		Deployment actually has the same problem, however it is possible to bypass this issue for Deployment by setting
+		minReadySeconds for replicaset separately; unfortunately this workaround seems not work for cloneset
+
+		// if rollout.Spec.Strategy.IsBlueGreenRelease() && revision.IsContinuousRelease(h.Client, oldObj, newObj) {
+		// 	err = fmt.Errorf("successive release is not supported for bluegreen for now, rollback first")
+		// 	return false, fmt.Errorf(err.Error())
+		// }
+	*/
+
```

Comment:
plz move the comment to the controller logic

Reply:
done